test/run_test.sh

```diff
@@ -10,7 +10,7 @@ ROOT_DIR=$TEST_DIR/..
 trap cleanup EXIT
 cleanup() {
     cd $ROOT_DIR
-    node index.js stop all
+    #node index.js stop all
```

Comment:
is this file still used anywhere? Looks we can delete it

Reply:
It's for local testing. Let's keep it for now.